test/unit.js

```diff
@@ -26,7 +26,62 @@ describe('bore', () => {
     expect(<Fn />.localName).to.equal('div');
   });
 
-  it('setting attributes', () => {
+  it('setting attributes on CustomElement', () => {
```

Comment:
> also because non obvious naming of attrs test1 and so on

Not sure if I understand that statement.

If it's already being tested, it'd be good if we can just remove this test. It's quite complex and difficult to understand.